src/olympia/hero/admin.py

```diff
@@ -30,7 +30,6 @@ class Media:
     model = PrimaryHero
     fields = (
         'description',
-        'image',
```

Comment:
You can drop the field from the model too.   (but we don't want to actually remove the database field until a following deploy to production.  steps:
- add `null=True` kwarg to `image` in `PrimaryHero`
- make migrations
- delete `image` from `PrimaryHero`
- (a few days later so it's not in the same tag;) run makemigrations again to actually delete the field.

Reply:
> * add `null=True` kwarg to `image` in `PrimaryHero`
> * make migrations
> * delete `image` from `PrimaryHero`
> * (a few days later so it's not in the same tag;) run makemigrations again to actually delete the field.

Should I migrate after the first `makemigrations`?

Reply:
yes.  The last step can be done whenever - we're pretty lax about cleaning up the database once it's not referenced in the code.